src/box/memtx_engine.cc

```diff
@@ -70,6 +70,7 @@ static void
 replica_join_cancel(struct cord *replica_join_cord);
 
 enum {
+	TUPLE_ARENA_SIZE_MIN = 32 * 1024 * 1024,
```

Comment:
There must be no such constant. The limit of 32MB is not arbitrary, it depends on other constant(s) and features. We should reveal causes of such behavior.

Reply:
Done.